packages/accordion/package.json

```diff
@@ -34,15 +34,15 @@
   ],
   "dependencies": {
     "@polymer/polymer": "^3.0.0",
-    "@vaadin/component-base": "^23.0.1",
-    "@vaadin/details": "^23.0.1",
-    "@vaadin/vaadin-lumo-styles": "^23.0.1",
-    "@vaadin/vaadin-material-styles": "^23.0.1",
+    "@vaadin/component-base": "^23.1.0-alpha0",
+    "@vaadin/details": "^23.1.0-alpha0",
+    "@vaadin/vaadin-lumo-styles": "^23.1.0-alpha0",
+    "@vaadin/vaadin-material-styles": "^23.1.0-alpha0",
     "@vaadin/vaadin-themable-mixin": "^23.0.1"
```

Comment:
Themable mixin version isn't bumped in many files

Reply:
Updated manually. Something went wrong with a script 🤔